tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -326,6 +328,94 @@ def test_login_with_existing_user(mocker, api_client, user):
     assert response.status_code == status.HTTP_302_FOUND
 
 
+@pytest.mark.django_db
+def test_login_with_old_email(mocker, api_client, user):
```

Comment:
This tests that user's who try to login with an old email then have their email updated with the one in the `decoded_token` from the auth payload.